docs/license.rst

```diff
@@ -25,13 +25,24 @@ documentation.
 -   "AUTHORS" hereby refers to all the authors listed in the
     :ref:`authors` section.
 
--   The ":ref:`babel-license`" applies to all the sourcecode shipped as
+-   The ":ref:`babel-license`" applies to all the source code shipped as
     part of Babel (Babel itself as well as the examples and the unit tests)
     as well as documentation.
 
+-   The ":ref:`unicode-license`" applies to the transformed Unicode
+    Common Locale Data Repository (CLDR) data files shipped with Babel,
+    in the directory ``babel/locale-data``.
+
 .. _babel-license:
 
 Babel License
 -------------
 
 .. include:: ../LICENSE
+
+.. _unicode-license:
+
+Unicode License
+---------------
+
+.. include:: ../babel/localedata/LICENSE.unicode
```

Comment:
I have not tested it, but shouldn't it be `locale-data` here as well?

Reply:
It absolutely should, great catch. Thanks!